rocksdb_admin/tests/CMakeLists.txt

```diff
@@ -17,6 +17,10 @@ endforeach(testsourcefile ${TEST_SOURCES})
 
 add_executable(hbase_backup_integration_test hbase_backup_integration_test.cpp)
 target_link_libraries(hbase_backup_integration_test gtest rocksdb_admin boost_filesystem rocksdb_glogger kafka_consumer rdkafka++)
+# uncomment following line to include the integration test
+#add_test(NAME hbase_backup_integration_test COMMAND hbase_backup_integration_test)
```

Comment:
Any reason for the tests being commented out?

Reply:
Yes, the integration test will indeed upload data to our s3 bucket. Thus, we need comment it out for package. But, for local actual test, we need to uncomment to run the test.